Controller/ApiController.php

```diff
@@ -40,9 +36,6 @@ class ApiController
     /** @DI\Inject("jms_translation.config_factory") */
     private $configFactory;
 
-    /** @DI\Inject */
-    private $request;
-
```

Comment:
Yeah, I saw that. The reason I made them here was because the tests failed otherwise